### PR TITLE
[python-termcolor] [ubuntu] wildcard for python2 and trusty-cosmic for python3

### PR DESCRIPTION
**Pull request by veimox**
- https://packages.ubuntu.com/bionic/python-termcolor
- https://packages.ubuntu.com/bionic/python3-termcolor

- [x] rosdistro_reformat
- [x] nosetests

### CODE REVIEW

**Comment by tfoote** [comment]
Same here please add zesty, artful and cosmic.

File: rosdep/python.yaml [hunk 1]
```diff
@@ -3812,13 +3812,13 @@ python-termcolor:
     pip:
       packages: [termcolor]
   ubuntu:
-    trusty: [python-termcolor]
+    '*': [python-pyudev]
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: Copy-n-paste: this should be `python-termcolor`

**Review by mikaelarguedas — CHANGES_REQUESTED** [review]

**Comment by mikaelarguedas** [comment]
> is just a coincidence or is there something against debian unstable?

As `sid` is a moving target it makes keeping the database up-to-date challenging.
From [contribution guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules):

`If there's a new distro in prerelease adding a rule for it is fine. However please don't target 'sid' as it's a rolling target and when the keys change our database gets out of date.`

**Review by mikaelarguedas — APPROVED** [review]
lgtm thanks @veimox for iterating

**Pull request merged**